FIx broken file links for development files

sirlupusdev opened the pull request:
There's not much to say here, just repairing some file links after development files were moved from .dev to docker/development.

rihards-simanovics reviewed (approved):
hi @sirlupusdev, thanks.

lgtm

sirlupusdev commented:
Found a few other things, will add to this PR.

rihards-simanovics commented:
> Found a few other things, will add to this PR.

If there is more, it's best to mark this as a draft so it doesn't get picked up for review prematurely. Thanks again for the help!

sirlupusdev commented:
I've applied the changes done to the mysql compose files to the other files as well. 

There is no invoiceshelf container (anymore), and the php container which has access to artisan doesn't have bash, so I've updated that command. Additionally, the db-pgsql and db-mysql aliases for connecting to the database don't seem to exist (anymore in new builds), so I've replaced it with the IP. The build and setup/installation wizard process of all compose files has been tested. 

The PR is now ready for merge.
